packages/http-client-java/emitter/src/common/client.ts

```diff
@@ -70,3 +78,16 @@ export class ServiceVersion extends Metadata {
 export interface EncodedSchema {
   encode?: string;
 }
+
+export class PageableContinuationToken {
+  parameter: Parameter;
```

Comment:
nit; for someone new reading the code, can we have a comment on what is `parameter` member? - IIUC, it's reference to the request query or header parameter representing the continutation token to send to the service, right?

Reply:
Agree. I've not doing good doc in this lib.

I've added to this class and some more. I will put more doc (and hopefully UT) to the lib in future.